src/layout/index.tsx

```diff
@@ -15,12 +15,7 @@ const AppLayout: React.FC<Props> = () => {
 
   return (
     <>
-      {/* //INFO Sander -> ik wou het woord Prijslijst bij in die header, dus voorlopig op deze manier opgelost.
-    title wordt meegegeven aan appheader, als die string leeg is dan staat er gewoon "De Spelmaeckers", anders komt de string er bij.
-
-    als er ooit nog extra pagina's bijkomen is dit uiteraard niet de oplossing...
-    */}
-      {!isHomePage && <AppHeader title="Prijslijst" />}
+      {!isHomePage && <AppHeader title="header.menuPage" />}
```

Comment:
@matthias-D  Inderdaad niet future proof maar hier zou ik nu nog niet wakker van liggen. Ik ben nog niet overtuigd van die layout structuur

Reply:
mag er in principe uit, want on mobile is er toch nie genoeg plek om "De Spelmaeckers | Prijslijst" te zetten... :D

Reply:
Tenzij ik on mobile die header hoog genoeg maak om 2 lijnen aan copy te voorzien